### DISCUSSION TITLE
Set custom fields as unique value

### DISCUSSION THREAD

**Discussion by mabdelgaliel** [netbox-community/netbox#6067] [Q&A]
Hi, is there anyway to set a custom field to accept only unique values, so users cannot assign a duplicate value in that field?

Thanks in advance,

**Comment by xcdr** [comment]
This feature is also important for me, but as I know there isn't any option.

**Comment by gellis713** [comment]
@jeremystretch Is there any fundamental issue with this concept or does it go against the principles of Netbox? If not I'd be willing to give this contribution a go.

**Reply by farfigneugan** [reply]
This is already implemented for IPAM; ip-addresses entries. It would be useful to include a check for duplicate MAC addresses (for example)

**Reply by PieterL75** [reply]
If it is an optional knob when creating a custom field, then I like it.
